Sorstr.py

```diff
@@ -1,55 +0,0 @@
-from glob import glob
-
-import re
-
-
-class Sorstr:
-    inverted_index = {}
-
-    def index(self, pattern: str) -> None:
-        """
-        Index files indicated by the pattern.
-
-        :param pattern: The pattern to use to look for files 
-        """
-        self.inverted_index = {}
-
-        files = glob(pattern)
-        files.sort()
-        for file in files:
```

Comment:
`file` is a type in the standard library. While using it as a variable name is no problem if it's not imported, it's a risk.